packages/template-retail-react-app/app/components/product-tile/index.test.js

```diff
@@ -45,6 +45,18 @@ test('Renders Skeleton', () => {
     expect(skeleton).toBeDefined()
 })
 
+test('Renders PricingAndPromotionsSkeleton when isRefetching is true', async () => {
+    const {getAllByRole, getByTestId, queryAllByText, queryByTestId} = renderWithProviders(
+        <ProductTile isRefreshingData={true} product={mockMasterProductHitWithMultipleVariants} />
+    )
+
+    expect(getByTestId('product-tile-image')).toBeDefined()
+    expect(getAllByRole('radio')).toBeDefined()
+    expect(queryAllByText(/£18\.55/i)).toHaveLength(0)
+    expect(queryAllByText(/£31\.36/i)).toHaveLength(0)
+    expect(queryByTestId('promo-callout')).not.toBeInTheDocument()
```

Comment:
This is a bit hacky, and error prone. The best is to assert the exact skeleton element. Imagine if someone tries to put a promo-call somewhere in the tile without the skeleton. This test will fail. Why don't we put a data-testid in the `PricingAndPromotionsSkeleton` and assert that block?